src/Microsoft.Diagnostics.Monitoring.EventPipe/Counters/EventPipeCounterPipelineSettings.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Diagnostics.Monitoring.EventPipe
+{
+    public class EventPipeCounterPipelineSettings : EventSourcePipelineSettings
+    {
+        public EventPipeCounterGroup[] CounterGroups { get; set; }
+        public TimeSpan RefreshInterval { get; set; }
```

Comment:
Does it make sense to move `RefreshInterval` to `EventPipeCounterGroup` since that's where it will be defined when converting this information to EventPipeProvider? I'm thinking it should be this way so that we don't block a scenario where someone wants to specify different counter groups with different intervals.

Reply:
I'll make a separate issue for this.